content/source/docs/enterprise/api/policies.html.md

```diff
@@ -205,6 +205,8 @@ This endpoint uploads code to an existing Sentinel policy.
 
 -> **Note**: This endpoint does not use JSON-API's conventions for HTTP headers and body serialization.
 
+-> **Note**: This endpoint limits the size of uploaded policies to 10MB. If a larger payload is uploaded, an HTTP 413 error will be returned, and the policy will not be saved.
+
```

Comment:
I wonder if it's worth providing a suggestion to refactor the policy into multiple smaller policies?
Otherwise, looks good.

Reply:
Added!